app/helpers/course/virtual_classrooms_helper.rb

```diff
@@ -7,4 +7,13 @@ module Course::VirtualClassroomsHelper
   def virtual_classrooms_title
     @settings.title.nil? ? nil : format_inline_text(@settings.title)
   end
+
+  # Generate select options for virtual classroom duration
+  #
+  # @return [[String, Integer]] Formatted select options, starting from 15 minutes,
+  #   with a 15-minute interval, up to max_duration (inclusive)
+  #   set in virtual_classroom component settings
+  def duration_options
+    (15..(@settings.max_duration.to_i).to_i).step(15).map { |i| ["#{i} minutes", i] }
```

Comment:
@xyc0562 any reasons that the `to_i` is used twice here? Also the "minutes" string need to be translated.

LGTM otherwise.